modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -94,6 +103,59 @@ asmctl api create
 The API that was created is displayed.
 . Optionally, you can verify that an asset is created on Exchange and an API is created on API Manager.
 
+[[create-api-ASM-CLI-ConnAps]]
+=== Autocreate by Using Your Connected Apps Credentials
```

Comment:
```suggestion
=== Authenticate Users Managed by an IdP
```

Reply:
Autocreate APIs for Users Managed by an IdP

Done! Thanks!